src/core/resources/WallContentWindow.qml

```diff
@@ -11,6 +11,32 @@ BaseContentWindow {
         anchors.fill: parent
     }
 
+    Rectangle
+    {
+        visible: options.showWindowTitle
+        y: -Style.windowTitleHeight + ( parent.border.width / 2 )
+        x: 0
+        width: parent.width
+        height: Style.windowTitleHeight - parent.border.width
+        color: parent.border.color
+        border.width: parent.border.width
+        border.color: parent.border.color
+
+        Text
+        {
+            // y delta relative to title rectangle height ( 8 is the best value )
+            y: Style.windowTitleHeight / 8
+            // x delta relative to font size ( 4 is the best value )
+            x: Style.windowTitleFontSize / 4
+            FontLoader { id: gothamBook; source: "../../Gotham-Book.otf"; name: "qrc::gotham-book" }
```

Comment:
Absolue path for source instead of relative? Is one loader for each Text ok or should it be done only in one place?